src/htmx.js

```diff
@@ -1235,13 +1235,12 @@ return (function () {
         }
 
         function consumeCSSSelector(tokens) {
-            var result = "";
-            if (tokens.length > 0 && tokens[0].match(COMBINED_SELECTOR_START)) {
+            if (tokens.length > 0 && COMBINED_SELECTOR_START.test(tokens[0])) {
                 tokens.shift();
-                result += consumeUntil(tokens, COMBINED_SELECTOR_END).trim();
+                var result = consumeUntil(tokens, COMBINED_SELECTOR_END).trim();
                 tokens.shift();
             } else {
-                result += consumeUntil(tokens, WHITESPACE_OR_COMMA);
+                var result = consumeUntil(tokens, WHITESPACE_OR_COMMA);
             }
```

Comment:
Mhh, that looks wrong to me. i meant:
```js
var result;
if (...) {
  result = consumeUntil(...)
} else {
  result = consumeUntil(...)
}
return result;
```

Reply:
Right, that makes more sense to me as well. Fixed it in the force push below.